pulp_ansible/app/viewsets.py

```diff
@@ -260,6 +596,55 @@ class AnsibleRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin):
     queryset = AnsibleRepository.objects.all()
     serializer_class = AnsibleRepositorySerializer
 
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "create",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:ansible.add_ansiblerepository",
```

Comment:
This condition should be `has_model_perms`. When I tried this out with pulp_file I got a 500 error when trying to create a repo. Also, when testing locally you have to run `pulpcore-manager migrate` to ensure any updates to the access policy get updated in the db, we have a hook that checks for updates.